static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -0,0 +1,210 @@
+.new-header {
+    background-color: colour(guardian-brand);
+}
+
+.new-header__inner {
+    display: flex;
+    justify-content: space-between;
+
+    @include mq($until: phablet) {
+        align-items: flex-start;
+
+        // On mobile we want the nav to appear on a new line.
+        // This is achieved by allowing flex items to wrap, and placing a full
+        // width item after the logo.
+        // In the future this will be better achieved using display: contents
+        // http://stackoverflow.com/questions/29732575/line-break-in-multi-line-flexbox
+        flex-wrap: wrap;
+
+        &::after {
+            content: '';
+            width: 100%;
+        }
+    }
+
+    @include mq(phablet) {
+        align-items: center;
+        // TODO: Or increase padding on target areas instead?
+        box-sizing: border-box;
+        padding-right: ($gs-gutter / 2);
+        padding-left: ($gs-gutter / 2);
+    }
+}
+
+.new-header__logo {
+    padding: ($gs-baseline / 2) ($gs-gutter / 2);
+
+    @include mq(phablet) {
+        order: 1;
+    }
+
+    svg {
+        // Removes extra spacing
+        display: block;
+
+        @include mq(tablet) {
+            width: 200px;
+            height: auto;
+        }
+    }
+}
+
+.new-header__edition-button {
+    @include fs-textSans(2);
+    order: -1;
+    color: colour(news-support-1);
+    // Override from fs-textSans mixin
+    line-height: 1;
+    // TODO: Use reset util?
+    // Unset button from user agent stylesheet
```

Comment:
I plan to remove these override/unset comments, but they are useful for me whilst I continue to iterate on this.